relay-general/src/pii/processor.rs

```diff
@@ -195,6 +198,78 @@ impl<'a> Processor for PiiProcessor<'a> {
         replay.process_child_values(self, state)?;
         Ok(())
     }
+
+    fn process_event(
+        &mut self,
+        event: &mut Event,
+        _meta: &mut Meta,
+        state: &ProcessingState<'_>,
+    ) -> ProcessingResult {
+        // TODO: make keys lazy init
+        let mut keys = Vec::new();
+
+        // collect the variables keys and scrub them out
+        if let Some(request) = event.request.value_mut() {
+            if let Some(Value::Object(data)) = request.data.value_mut() {
+                if let Some(Annotated(Some(Value::Bool(graph_ql_error)), _)) =
+                    request.other.get("graphql_error")
```

Comment:
Still have to decided where to place the marker `graphql_error`

Reply:
When we do, please make sure it's typed, so we don't have to read from `request.other` (which is only for forwarding data).